src/map/packets_struct.h

```diff
@@ -401,6 +389,60 @@ DEFINE_PACKET_ID(ZC_PAR_4JOB_CHANGE, 0x0b25);
 #pragma pack(push, 1)
 #endif // not NetBSD < 6 / Solaris
 
+struct PACKET_ZC_PAR_CHANGE {
+	int16 PacketType;
+	uint16 varID;
+	int32 count;
+} __attribute__((packed));
+DEFINE_PACKET_HEADER(ZC_PAR_CHANGE, 0x00b0);
+
+struct PACKET_ZC_LONGPAR_CHANGE {
+	int16 PacketType;
+	uint16 varID;
+	int32 amount;
+} __attribute__((packed));
+DEFINE_PACKET_ID(ZC_LONGPAR_CHANGE, 0x00b1);
+
+struct PACKET_ZC_STATUS_CHANGE {
+	int16 PacketType;
+	uint16 statusID;
+	uint8 value;
+} __attribute__((packed));
+DEFINE_PACKET_HEADER(ZC_STATUS_CHANGE, 0x00be);
+
+struct PACKET_ZC_NOTIFY_CARTITEM_COUNTINFO {
+	int16 PacketType;
+	int16 curCount;
+	int16 maxCount;
+	int32 curWeight;
+	int32 maxWeight;
+} __attribute__((packed));
+DEFINE_PACKET_HEADER(ZC_NOTIFY_CARTITEM_COUNTINFO, 0x0121);
+
+struct PACKET_ZC_ATTACK_RANGE {
+	int16 PacketType;
+	int16 currentAttRange;
+} __attribute__((packed));
+DEFINE_PACKET_HEADER(ZC_ATTACK_RANGE, 0x013a);
+
+struct PACKET_ZC_COUPLESTATUS {
+	int16 PacketType;
+	uint32 statusType;
+	int32 defaultStatus;
+	int32 plusStatus;
+} __attribute__((packed));
+DEFINE_PACKET_ID(ZC_COUPLESTATUS, 0x0141);
+
+#if PACKETVER_MAIN_NUM >= 20170906 || PACKETVER_RE_NUM >= 20170830 || defined(PACKETVER_ZERO)
+// TODO: Unconfirmed name
+struct PACKET_ZC_LONGLONGPAR_CHANGE {
+	int16 PacketType;
+	uint16 varID;
+	int64 amount;
+} __attribute__((packed));
+DEFINE_PACKET_ID(ZC_LONGLONGPAR_CHANGE, 0x0acb);
```

Comment:
here name should be ``ZC_LONGLONGPAR_CHANGE``

Reply:
You want me to remove the TODO?
Because the name is exactly like you wrote it

Reply:
ah yes sorry, i not saw what name is correct already

Reply:
@Lemongrass3110 you can remove the TODO the name is confirmed.

Reply:
Not really confirmed, but at least it seems to match the disassembly. :-)

Reply:
It is confirmed :)

Reply:
Leaky leaky?
Guessing from function name is not confirming real names.